doc-High_Availability_Guide/topics/Renaming_HA_appliances.adoc

```diff
@@ -0,0 +1,130 @@
+[[renaming-ha]]
+== Renaming Database-Only Appliances
+
+Renaming database-only appliances in a high availability setup must be performed in a specific order to maintain database replication. After changing the hostnames of the database-only appliances, you will need to re-configure high availability on the primary and standby database-only appliances.
+
+
+=== Updating Hostnames on Database-Only Appliances
+
+. Confirm and note the hostnames of the active database (primary) appliance and the standby database appliance in the appliance console. 
++
+You can verify this from the appliance summary screen of a database-only appliance, where the status for *Local Database Server* shows _primary_ or _standby_, for example:
++
+----
+...
+Local Database Server:   running (primary)
+...
+----
++
+. Stop the failover monitor on the non-database {product-title_short} appliances:
+.. In the appliance console menu, select *Configure Application Database Failover Monitor*. 
+.. Select *Stop Database Failover Monitor*.
+. Stop the database service on the standby database-only appliance:
++
+----
+# systemctl stop rh-postgresql95-postgresql
+----
++
+. Stop `evmserverd` on each {product-title_short} (non-database) appliance:
++
+----
+# systemctl stop evmserverd
+----
++
+. Rename the database-only appliance and restart that appliance:
+.. In the appliance console, configure the new hostname or IP address from the *Configure Network* option.
+.. Restart the renamed appliance.
+//Rename, then restart the standby if desired now? [Dayle: need to confirm the goal with bug reporter.]
++
+. On all appliances, change the host key to the primary database server's new hostname in `vmdb/config/database.yml`.
```

Comment:
After adding appropriate `hostname`, `Postgresql` service need to start or after this line it should point to Next step i.e. re-configuring Primary or Standby DB-only appliances.?

Reply:
I am still wondering this too.

Reply:
@dayleparker After changing hostname, you have to start first postgresql service and then you will have to start evmserverd service.

Reply:
Excellent, thank you for confirming, Satyajit.